app/scenes/load_team/load_team.js

```diff
@@ -25,7 +25,8 @@ export default class LoadTeam extends PureComponent {
     };
 
     static navigationProps = {
-        renderBackButton: () => null
+        renderBackButton: () => null,
+        renderTitleComponent: () => <View style={{flex: 1, backgroundColor: 'blue'}}/>
```

Comment:
what is this blue here? well I don't really see any blue while testing so im going to approve